src/python/WMCore/Algorithms/MathAlgos.py

```diff
@@ -117,13 +115,12 @@ def createHistogram(numList, nBins, limit):
         # Nothing to do?
         return histogram
 
-
     histEvents.sort()
     upperBound = max(histEvents)
     lowerBound = min(histEvents)
     if lowerBound == upperBound:
         # This is a problem
-        logging.error("Only one value in the histogram!")
+        logging.debug("Only one value in the histogram!")
```

Comment:
If it's really a problem as the comment says, then we should make it a warning at least :)

Reply:
I am not sure whether this is the problem, next code basically handles this cases. So not sure whether this need to warned.